testproject/testproject/urls.py

```diff
@@ -1,27 +1,26 @@
-from django.conf.urls import patterns, include, url
+from django.conf.urls import include, url
 from django.conf import settings
 from django.contrib.staticfiles.urls import staticfiles_urlpatterns
+from django.views.static import serve as static_serve
 
 from django.contrib import admin
 admin.autodiscover()
 
-urlpatterns = patterns('',
-    url(r'^admin/doc/', include('django.contrib.admindocs.urls')),
+urlpatterns = [
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^notify/', include('django_notify.urls', namespace='notify')),
```

Comment:
This "extra" notify pattern caused a conflict in 1.11 and had been removed on the upstream branch (with a note that it was extra).